src/sandwich-menu/containers/hooks/use-on-add-cart.js

```diff
@@ -0,0 +1,13 @@
+import {useDispatch, useSelector} from "react-redux";
+import {addItemToShoppingCart} from "../../../shopping-cart/actions"
+
+export const useOnAddCart = (selected, setSelected, selector) => {
```

Comment:
This hook looks ...smelly. Let's move the `selected`, `setSelected` pair to the sandwich context.

![image](https://user-images.githubusercontent.com/23569938/109505733-956be200-7aa5-11eb-8b94-1331f474d9b3.png)

Also this mapping can be done inside the reducer.

Reply:
Done

Reply:
Done